### PR TITLE
Add some more explanations for FVA and add `flux_summary` usage to notebook

### LINKED ISSUE #345: Explain FVA better
My undergrad is struggling with FVA, especially all the options. I think we should explain it more carefully 🚸 in the docs

### PR DESCRIPTION
**Pull request by stelmo**
Should close #345 once done.

### CODE REVIEW

File: docs/src/notebooks/2_finding_balance.jl [hunk 1]
```diff
@@ -86,6 +86,10 @@ dict_soln = flux_balance_analysis_dict(
     ],
 )
 
+# This solution can be display using `flux_summary`. Note, this pretty printing only works
```
Comment (exaexa) [inline docs/src/notebooks/2_finding_balance.jl:89]: "Note" is ....notish. :D

**Review by exaexa — APPROVED** [review]
lgtm

File: docs/src/notebooks/2_finding_balance.jl (hunk 1 above)
Comment (stelmo) [reply docs/src/notebooks/2_finding_balance.jl:89]: ha ha noted :]

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit bd860ebf is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 07b2f494 is formatted properly.

**Pull request merged**